redisson/src/main/java/org/redisson/api/RAtomicLongAsync.java

```diff
@@ -48,6 +48,13 @@ public interface RAtomicLongAsync extends RExpirableAsync {
      * @return the updated value
      */
     RFuture<Long> decrementAndGetAsync();
+    /**
+     * Atomically decrements the given value to the current value.
+     * 
+     * @param delta the value to decrements
+     * @return the updated value
+     */
+    RFuture<Long> decrementAndGetAsync(long delta);
```

Comment:
This method should be also added to RAtomicLongRx and RAtomicLongReactive interfaces